src/shared/schemas.ts

```diff
@@ -23,9 +23,9 @@ import { getStringHash } from './utilities/textUtilities'
 const goformationManifestURL = 'https://api.github.com/repos/awslabs/goformation/releases/latest'
 const devfileManifestURL = 'https://api.github.com/repos/devfile/api/releases/latest'
 const schemaPrefix = `${AWS_SCHEME}://`
-const hostedFilesCloudFrontDistribution = 'https://d3rrggjwfhwld2.cloudfront.net'
-const hostedFilesS3Fallback = 'https://aws-vs-toolkit.s3.amazonaws.com'
-const buildSpecHostedFilesPath = '/CodeBuild/buildspec/buildspec-standalone.schema.json'
+const buildspecHostedFilesPath = '/CodeBuild/buildspec/buildspec-standalone.schema.json'
+export const buildspecCloudfrontURL = 'https://d3rrggjwfhwld2.cloudfront.net' + buildspecHostedFilesPath
+export const buildspecS3FallbackURL = 'https://aws-vs-toolkit.s3.amazonaws.com' + buildspecHostedFilesPath
```

Comment:
For my own curiosity: why does an S3 fallback URL exist in addition to CF? Seems excessive, I'd at least hide this behind an abstraction if it's a pattern.

Reply:
good callout. This seems more confusing than useful.

-  adds a direct dependency to multiple names for the same thing. 
-  doubles the size of the config that we need to maintain.

however this decision was made before this PR, so probably should not change it now.

Reply:
I just removed the S3 fallback, I was just checking some documentation and the s3 bucket will be phased out over time. The original reason was just for extra availability just in case cloudfront went down. If I'm not mistaken, the visual studio toolkit will crash if it can't find certain files, that's why they were using that approach (and I followed that with our implementation). After double checking on VSCode though, we don't have this problem since we handle schema download failures